src/components/content/heroes/aurora/weather-easter-egg/snow.tsx

```diff
@@ -0,0 +1,33 @@
+import PrecipitationEffect, { PrecipitationType } from './precipitation-effect';
+import React, { FC } from 'react';
+import { DefaultFlares, DefaultFlaresDark } from '../default-flares';
+import styled from 'styled-components';
+
+export const AuroraSnowyFlareColor = styled.div`
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: linear-gradient(0deg, #4943cb99 0%, #4d79ff00 90%);
+  position: absolute;
+`;
+
+interface SnowProps {
+  amount: number;
+}
+
+export const Snow: FC<SnowProps> = ({ amount }) => {
```

Comment:
Seeing `React.FC` here I want to let you know that it's heavily discussed if you should use it or not.

Take this article and then start digging:
https://juhanajauhiainen.com/posts/should-you-use-reactfc-to-type-your-components

I'm still always torn, that's why it's more a FYI